openmetadata-ui/src/main/resources/ui/src/pages/ExplorePage/ExplorePageV1.component.tsx

```diff
@@ -407,10 +421,13 @@ const ExplorePageV1: FunctionComponent = () => {
           showErrorToast(error);
         }
       })
-
       .finally(() => setIsLoading(false));
   };
 
+  useEffect(() => {
+    setSidebarActiveTab(ExploreSidebarTab.ASSETS);
+  }, []);
```

Comment:
@karanh37, what do you think about setting the default active tab to 'Asset' so that we can avoid writing this useEffect?

Reply:
It's default set to 'Asset' in store, but if we change it to tree store is updated.
To reset the store kept it like this